javascript/javascript2/week1/name-generator/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="nameGen.js"></script>
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <input id="user-input" placeholder="Write your name here...">
+        <button id="button" onclick="btnClicked()" style="display: block;">Accept</button>
+        <p id="displayed-text"></p>
+        <br>
+        <button id="sec-button" onclick="secBtnClicked()" style="display: none;">One more spirit please</button>
```

Comment:
Please use `addEventListener` in the javascript to add an event to a dom element. intead of in the html